client/my-sites/checkout/checkout/style.scss

```diff
@@ -1109,7 +1109,7 @@
 	.formatted-header {
 		color: var( --color-jetpack-onboarding-text );
 	}
-	
+
```

Comment:
My auto prettier formatter is removing the tab here.

Reply:
Good riddance if you ask me!